content/docs/tutorials/fullstack.md

```diff
@@ -11,15 +11,15 @@
 
 **Coming Soon (August 2022)**
 
-Create a site to help schedule tennis matches to learn how Cdev can be used to create and manage full stack web applications.
+Create a site to create and keep diary entries to learn how Cdev can be used to create and manage full stack web applications.
 
 
 In this tutorial, we will be going through the entire process of creating a full stack application with Cdev. We will be focusing on demonstrating how Cdev can integrate with standard python tools and other development workflows to create a great developer experience. We will explain all the components and steps of this tutorial in depth, but it does help to have some familiarity with some of the technologies and concepts around full stack development.
```

Comment:
Capitalize 'Python'

Reply:
'to create a great developer experience' I think we could remove this bit. I think it is true, but I have seen people comment on other tools that they should not contain 'qualified' statements.